src/ResourceManager/Media/Commands.Media/Generated/IMediaServiceOperations.cs

```diff
@@ -0,0 +1,174 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
```

Comment:
Please Create a management client in azure-sdk-for-net and reference the management client in these cmdlets.  You should not include the generated code directly here.